ruby_on_rails/app_initialisation.md

```diff
@@ -13,7 +13,11 @@
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
 You can do this with `gem update rails`. Beware of beta versions.
 
-* Start a new Rails project using `rails new [project-name] --database=postgresql --no-skip-test --skip_action_mailbox` where the `project-name` is exactly the one you chose before.
+* Start a new Rails project using
+```
+rails new [project-name] --database=postgresql --skip_ci --no-skip-test --skip_action_mailbox -m https://raw.githubusercontent.com/renuo/applications-setup-guide/main/ruby_on_rails/template.rb
```

Comment:
```suggestion
rails new [project-name] --database=postgresql --skip-ci --skip-keeps --skip-test --skip-action-mailbox -m https://raw.githubusercontent.com/renuo/applications-setup-guide/main/ruby_on_rails/template.rb
```

`--skip-test` because we use rspec instead of `/tests/` dir
`--skip-keeps` because it's useless

Reply:
I don't see the harm in having the `.keep` files

Reply:
I agree, but let's not make this PR too big. I am keen in adding those options, but not here